### PR TITLE
update sample to use passport-azure-ad 3.0.0

### PR DESCRIPTION
**Pull request by lovemaths**
Will merge after the release of version 3.0.0.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
+    "assert-plus": "*",
+    "body-parser": "^1.15.2",
+    "bunyan": "*",
+    "connect-mongo": "^1.3.2",
+    "cookie-parser": "^1.4.3",
+    "ejs": ">= 0.0.0",
+    "ejs-locals": ">= 0.0.0",
+    "express": "3.x",
+    "express-session": "^1.14.1",
+    "method-override": "^2.3.6",
```
Comment (polita) [inline package.json:14]: What's this for?

File: package.json [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
+    "mongoose": "^4.6.2",
+    "passport": "*",
+    "passport-azure-ad": "^3.0.0",
+    "restify": "*"
```
Comment (polita) [inline package.json]: - seems dangerous. Could this break us?

**Comment by polita** [comment]
For the comments I made on the WebApp-OpenIDConnect-NodeJS PR that apply to all samples, please ensure we're making those changes across samples. Also, a couple new comments (which might also apply to the other samples). Afterwards, :shipit:

File: package.json (hunk 1 above)
Comment (lovemaths) [reply package.json:14]: It is used to allow the app to use http verbs. The app used to call express.(methodOverride()),  but this usage is deprecated. The functionality is moved to the method-override lib.

File: package.json (hunk 2 above)
Comment (lovemaths) [reply package.json]: Which one?

Comment (polita) [reply package.json]: Restify with a \* version.

**Pull request merged**